GitReader.Tests/Primitive/RepositoryTests.cs

```diff
@@ -211,6 +209,15 @@ public async Task GetRemoteUrls()
         await Verifier.Verify(repository.RemoteUrls);
     }
 
+    [Test]
+    public async Task GetRemoteUrls2()
+    {
+        using var repository = await Repository.Factory.OpenStructureAsync(
```

Comment:
I think this test intended to call `OpenPrimitiveAsync` API.

Reply:
Thanks, I will fix it!